mirrormanager2/templates/fedora/admin/master.html

```diff
@@ -1,94 +1,24 @@
 {% extends 'admin/base.html' %}
 {% import 'admin/lib.html' as lib with context %}
 {% import 'admin/static.html' as admin_static with context %}
+{% from "_macros.html" import footer, headincludes, nav, flashmessages, bottomincludes %}
```

Comment:
this template is why i used macros here -- cos the admin plugin basically needs its own master template, so used macros for the complex stuff here so we can re-use in the main master template